testing/TestHarness/TestHarness.Shared/App.xaml.cs

```diff
@@ -36,7 +36,10 @@ protected override async void OnLaunched(LaunchActivatedEventArgs args)
 		var host = UnoHost
 					.CreateDefaultBuilder()
 					.UseConfiguration(
-						configureHostConfiguration: builder => builder.AddSectionFromEntity(new LocalizationConfiguration { Cultures = new[] {"es", "en", "en-AU", "fr" } }))
+						configureHostConfiguration: (builder) =>
```

Comment:
This will need to be reverted - there's no appsettings.json file to load configuration from. The original code is there to make sure we have localization informatino loaded on app startup in the test harness